packages/xrpl/test/integration/utils.ts

```diff
@@ -444,3 +445,16 @@ export async function createAMMPool(client: Client): Promise<{
     asset2,
   }
 }
+
+export async function fetchAccountReserveFee(
```

Comment:
Instead of creating a new function, can we re-use the one from `packages/xrpl/src/sugar/autofill.ts` ? Its titled `fetchOwnerReserve`

This will improve the test-coverage of the function which is already in the main codebase.

Reply:
Hi Chenna, OwnerReserve and AccountReserve refers to 2 different fees. One would be `reserve_inc` and the other `reserve_base`.
Please refer to: https://xrpl.org/docs/references/protocol/ledger-data/ledger-entry-types/feesettings

Reply:
okay, thanks for the clarification. Personally, I'd have preferred the names `reserve_base` and `reserve_increment` because its easier to understand their intent. But its not a strong opinion.